packages/components/src/DateTimePickers/pickers/IncrementableScrollActionList/IncrementableScrollActionList.component.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import IncrementableScrollList from './IncrementableScrollList';
+import PickerAction from '../../PickerAction';
+
+class IncrementableScrollActionList extends React.Component {
+
+	constructor(props) {
+		super(props);
+
+		const middleInitialIndex = this.props.items.findIndex(
+			item => item.id === this.props.initialMiddleVisibleItemId);
+
+		this.initialIndex = middleInitialIndex === -1
+			? 0
+			: Math.max(middleInitialIndex - 2, 0);
+	}
+
+	isSelected(id) {
+		return id === this.props.selectedItemId;
+	}
```

Comment:
This method is useless, just inline it in the `render()` method

Reply:
It avoid creating new anonymous function for each row render call (when rerender component and also each scrolling)

Reply:
inlining `id === this.props.selectedItemId` doesn't create a function

Reply:
I wasn't thinking of this possibility, that's greater !